controller/userController.js

```diff
@@ -13,11 +17,29 @@ const login = (req, res) => {
         if (user) {
             const saltedPassword = SHA256(req.body.password + user.salt).toString();
             if (saltedPassword === user.passwordHash) {
-                const token = jwt.sign({ id: user._id }, AppConfig.SECRET, {
-                    expiresIn: 7200 // expires in 2 hours
+                // Generate JWT
+                const token = jwt.sign({ id: user._id }, AppConfig.SECRET);
+                // const token = jwt.sign({ id: user._id }, AppConfig.SECRET, { expiresIn: AppConfig.TOKEN_LIFESPAN });
+                // const refreshToken = jwt.sign({ id: user._id }, AppConfig.REFRESH_TOKEN_SECRET, { expiresIn: AppConfig.REFRESH_TOKEN_LIFESPAN });
+
+                // Password & salt should be used only during login
+                user.passwordHash = null;
+                user.salt = null;
+
+                // Generate response
+                const response = {
+                    auth: true,
+                    status: "Logged in",
+                    user: user,
```

Comment:
Consider using ES6 Object Property Shorthand. For example: user instead of user: user as object properties.

Reply:
@srkacha